about.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Creative Commons Global Summit 2015</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="icon" type="image/x-icon" href="/images/base/favicon.ico">
+</head>
+
+<body>
+    <nav class="top-menu">
+        <ul>
+            <li>
+                <a href="#"> <img src="images/base/facebook.png" alt="Facebook" /></a>
+            </li>
+            <li>
+                <a href="#"> <img src="images/base/twitter.png" alt="Twitter" /></a>
+            </li>
+            <li>
+                <a href="#">English</a>
+            </li>
+            <li>
+                <a href="#">My Page</a>
+            </li>
+             <li>
+                <a href="#">Logout</a>
+            </li>
+        </ul>
+    </nav>
+    <div id="toggle-menu" class="hidden">
+        <div class="close-wrapper clearfix"><img src="images/base/mobile/close.png" alt="Close" onclick="toggleMenu()" />
+        </div>
+        <nav class="toggle-nav">
+            <ul class="nav-list">
+
+                <li class="nav-item">
+                    <a href="about.html" class="nav-link">About</a>
+                </li>
```

Comment:
- [ ] In about page: make sure that the user can navigate from the about page to the home page by clicking on the `home` link in the nav bar (and also in the menu of the mobile version). I couldn't find a link that leads me to the home page.

<img width="205" alt="image" src="https://user-images.githubusercontent.com/77190203/157560934-7b2673a2-ae21-45a4-8ad8-680170e2b32b.png">

<img width="935" alt="image" src="https://user-images.githubusercontent.com/77190203/157560987-7b59530b-d274-40c9-bf12-accb0f092c0e.png">